Tests are not generated tests when custom header 'stdlib.h' exists in project's include path

## Steps to reproduce

1. Open and configure project `coreutils`
2. Generate tests for `lib/careadlinkat.c`
3. Check logs and see that compiler picks the project's `stdlib.h` file but not the system one.

```
2022-04-13 14:55:58.749 [utbot-ZZR7      ]      ShellExecTask.cpp:29       1| .   Execute: /utbot_distr/install/bin/clang -I/home/utbot/projects/coreutils/gnulib/lib -emit-llvm -c -Xclang -disable-O0-optnone -g -fstandalone-debug -fno-discard-value-names -fno-elide-constructors -DKLEE_MODE=1 -fsanitize=alignment,builtin,bounds,enum,float-cast-overflow,float-divide-by-zero,implicit-unsigned-integer-truncation,implicit-signed-integer-truncation,implicit-integer-sign-change,integer-divide-by-zero,nonnull-attribute,null,nullability-arg,nullability-assign,nullability-return,pointer-overflow,return,returns-nonnull-attribute,shift,unreachable,vla-bound -I/home/utbot/projects/coreutils/build/utbot/flags -c -I. -I.. -I./lib -Ilib -I../lib -Isrc -I../src -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -g -O0 -o /home/utbot/tmp/utbot-ZZR7/gnulib/lib/careadlinkat_klee.bc /home/utbot/tmp/utbot-ZZR7/gnulib/lib/careadlinkat_klee.c 
from directory: /home/utbot/projects/coreutils/build
2022-04-13 14:55:58.796 [utbot-ZZR7      ]      ShellExecTask.cpp:53       1| .   /home/utbot/tmp/utbot-ZZR7/gnulib/lib/careadlinkat_klee.c:80:11: error: no member named 'rpl_free' in 'struct allocator'
2022-04-13 14:55:58.796 [utbot-ZZR7      ]      ShellExecTask.cpp:53       1| .       alloc.free = *allocator_free_stub;
2022-04-13 14:55:58.796 [utbot-ZZR7      ]      ShellExecTask.cpp:53       1| .       ~~~~~ ^
2022-04-13 14:55:58.796 [utbot-ZZR7      ]      ShellExecTask.cpp:53       1| .   ./lib/stdlib.h:737:17: note: expanded from macro 'free'
2022-04-13 14:55:58.797 [utbot-ZZR7      ]      ShellExecTask.cpp:53       1| .   #   define free rpl_free
```

## Possible solution
Replace the following include in klee file
```c
#include <stdlib.h>
``` 
with something like
```c
#ifdef __cplusplus
extern "C"
#endif
void* calloc( size_t num, size_t size );
```
This trick should also reduce the time of preprocessing and as a consequence - compilation, but a bit.

--- a/server/src/printers/KleePrinter.cpp
+++ b/server/src/printers/KleePrinter.cpp
@@ -24,6 +24,11 @@ using namespace types;
 using printer::KleePrinter;
 
 static const std::string KLEE_GLOBAL_VAR_H = "klee_global_var.h";
+static const std::string CALLOC_DECLARATION = "extern\n"
+                                              "#ifdef __cplusplus\n"
+                                              "\"C\"\n"
+                                              "#endif\n"
+                                              "void* calloc(size_t num, size_t size);\n";
 
 printer::KleePrinter::KleePrinter(const types::TypesHandler *typesHandler,
                                   std::shared_ptr<BuildDatabase> buildDatabase,
@@ -76,9 +81,8 @@ fs::path KleePrinter::writeTmpKleeFile(
         strDeclareVar("int", PrinterUtils::KLEE_PATH_FLAG, "0");
     }
 
-    strInclude("klee/klee.h");
-    strInclude("stdlib.h", true);
-    ss << NL;
+    strInclude("klee/klee.h") << NL;
+    ss << CALLOC_DECLARATION << NL;
     writeStubsForStructureFields(tests);
 
     writeAccessPrivateMacros(typesHandler, tests, false);

